src/core/components/hv-route/index.tsx

```diff
@@ -605,4 +637,14 @@ export default function HvRoute(props: Types.Props) {
   );
 }
 
+export default function HvRoute(props: Types.Props) {
+  return (
+    <BackBehaviorProvider>
+      <HvRouteFC
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
```

Comment:
As a general practice, I can't recommend props spreading, and the linting rule is actually here for a good reason - it's easy with refactors to have unwanted props sneak in and cause bugs.